### PR TITLE
React router setup

### PR DESCRIPTION
**Pull request by sebastian-mereuta**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -3,3 +3,67 @@
...
+.rest {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.rest > * {
+  margin-left: 15px;
+}
+.active {
+  color: rgb(189, 83, 83);
```
Comment (AlexCondur) [inline src/App.css:38]: decide on a standard, use hex colors(#) by default and introduce rgba when dealing with opacity

File: src/App.css [hunk 2]
```diff
@@ -3,3 +3,67 @@
...
+  background-color: rgb(189, 83, 83);
+  border-radius: 2px;
+  transform: scale(1, 1);
+  transition: all 0.3s;
+}
+.activeNot {
+  color: rgb(189, 83, 83);
+  position: relative;
+}
+.activeNot::after {
```
Comment (AlexCondur) [inline src/App.css]: modify to hyphen-case when transitioning to .scss

File: src/App.tsx [hunk 3]
```diff
@@ -73,19 +73,18 @@ const App: React.FC = () => {
     return postData();
   }
 
-
-  useEffect(() => {
-    
-  }, []);
+  useEffect(() => {}, []);
```
Comment (AlexCondur) [inline src/App.tsx]: remove this

File: src/components/DropDown/DropDown.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        Log in
+      </NavLink>
+    </Menu.Item>
+    <Menu.Item key="1">
+      <a href="#">2nd menu item</a>
+    </Menu.Item>
+    <Menu.Divider />
+    <Menu.Item key="3">Log out</Menu.Item>
+  </Menu>
+);
```
Comment (AlexCondur) [inline src/components/DropDown/DropDown.tsx]: extract this into a separate component

File: src/components/DropDown/DropDown.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+//nu mai pot inchide dropdownul.
```
Comment (AlexCondur) [inline src/components/DropDown/DropDown.tsx]: de ce?

File: src/components/DropDown/DropDown.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Menu, Dropdown } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import { NavLink } from 'react-router-dom';
+import { Avatar } from 'antd';
+
+import { UserOutlined } from '@ant-design/icons';
+
+interface DropDownProps {}
```
Comment (AlexCondur) [inline src/components/DropDown/DropDown.tsx:9]: define

File: src/components/DropDown/DropDown.tsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    <Menu.Divider />
+    <Menu.Item key="3">Log out</Menu.Item>
+  </Menu>
+);
+
+//nu mai pot inchide dropdownul.
+
+const DropDown: React.FC<DropDownProps> = ({}) => {
+  return (
+    <div>
```
Comment (AlexCondur) [inline src/components/DropDown/DropDown.tsx]: do you need a wrapper div? if you are using this for layout purposes you could use items from antd layout(grid etc)

File: src/routes/Nav/Nav.tsx [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+import Dropdown from '../../components/DropDown/DropDown';
+
+interface NavProps {}
+
+const Nav: React.FC<NavProps> = ({}) => {
+  return (
+    <div>
+      <nav>
+        <div className="logo">
+          <NavLink className={({ isActive }) => (isActive ? 'active' : 'activeNot')} to="/">
```
Comment (AlexCondur) [inline src/routes/Nav/Nav.tsx]: use activeNot(with a different name) as default class and 'active' class optional

**Review by AlexCondur — APPROVED** [review]

File: src/components/DropDown/DropDown.tsx (hunk 4 above)
Comment (sebastian-mereuta) [reply src/components/DropDown/DropDown.tsx]: am mutat in componenta separata, dar isi pierde style-ul din antd. recomandarea de la antd e sa fie in aceeasi componenta, cum e mai sus

**Pull request merged**